compressor/conf.py

```diff
@@ -70,6 +70,12 @@ class CompressorConf(AppConf):
     OFFLINE_MANIFEST = 'manifest.json'
     # The Context to be used when TemplateFilter is used
     TEMPLATE_FILTER_CONTEXT = {}
+    # URL-less mode: you do offline compression & decompression regardless of settings.STATIC_URL
+    # and settings.COMPRESS_URL so your assets become re-locatable for any URLs.
+    # Affects manifest file contents and manifest key determination, but not the assets contents.
+    OFFLINE_URLLESS = False
```

Comment:
Yes. Sorry for the back-and-forth, but I think it's worth it to reduce complexity. For the record I still think the original issue is not a very common use-case, but after thinking about this for a while I don't see much downside in always doing the replace.